src/HttpHdrCc.cc

```diff
@@ -332,22 +357,16 @@ httpHdrCcStatDumper(StoreEntry * sentry, int, double val, double, int count)
 {
     extern const HttpHeaderStat *dump_stat; /* argh! */
     const int id = static_cast<int>(val);
-    const bool valid_id = id >= 0 && id < static_cast<int>(HttpHdrCcType::CC_ENUM_END);
-    const char *name = valid_id ? CcAttrs[id].name : "INVALID";
-
-    if (count || valid_id)
+    const auto name = ccNameByType(id);
+    if (count || name)
         storeAppendPrintf(sentry, "%2d\t %-20s\t %5d\t %6.2f\n",
-                          id, name, count, xdiv(count, dump_stat->ccParsedCount));
+                          id, name.value_or("INVALID"), count, xdiv(count, dump_stat->ccParsedCount));
```

Comment:
TIL about `value_or` as well

Reply:
?

Reply:
AFAICT, Francesco is simply acknowledging/celebrating that he has [learned something](https://www.howtogeek.com/702929/what-does-til-mean-and-how-do-you-use-it/). The first usage of value_or() in Squid code goes back to 2021, but things like that are easy to overlook in PR noise. I am glad this code cut through that noise!

Reply:
I was wondering what "TIL" stood for, have not seen that one before.